capstone_frontend/src/components/forms/Login.jsx

```diff
@@ -1,7 +1,170 @@
+import React from 'react';
+import Button from '@mui/material/Button';
```

Comment:
Good idea to use the Material ui library. We don't have to worry that much about css then.